src/openfermion/ops/_qubit_operator_test.py

```diff
@@ -55,6 +55,15 @@ def test_init_tuple(coefficient):
     assert qubit_op.terms[loc_op] == coefficient
 
 
+@pytest.mark.parametrize("coefficient", [0.5, 0.6j, numpy.float64(2.303),
```

Comment:
Since when are we using pytest? It is a reasonable idea to move everything over to pytest but I was under the impression everything was using unittest now. Is there any place in this code where pytest is used? If not I am somewhat inclined to suggest removing the pytest import and also this test. I think it is a reasonable goal to move over to pytest at some point (it has many advantages) but I think that is sort of its own "issue" and until that point I'd prefer that we remain consistent. But I am open to other thoughts about this. If there are other places you can point to in OpenFermion where we use pytest, I'll accept this.

Reply:
Most of this file uses the old pytest tests ;)